### PR TITLE
TRAD-230 Convert pools endpoint from POST to GET

### PR DESCRIPTION
**Pull request by yobloxroute**

### CODE REVIEW

**Review by DimaCantemir — APPROVED** [review]

File: examples/httpclient/main.go [hunk 1]
```diff
@@ -179,6 +180,21 @@ func callTradesHTTP() bool {
 	return false
 }
 
+func callPoolsHTTP() bool {
+	h := provider.NewHTTPTestnet()
+
+	pools, err := h.GetPools([]string{"Radium"})
```
Comment (aspin) [inline examples/httpclient/main.go:186]: @yobloxroute does this example work? Doesn't pools need to be `P_RAYDIUM`?

Comment (DimaCantemir) [reply examples/httpclient/main.go:186]: yeah, it's going to fail

Comment (aspin) [reply examples/httpclient/main.go:186]: I think what actually ends up happening here is that it gets ignored. And at the moment (until my PR gets merged in), Pools endpoint doesn't actually check the requested project list

Comment (yobloxroute) [reply examples/httpclient/main.go:186]: Will go through endpoints tests, thank you for pointing this out!

Comment (yobloxroute) [reply examples/httpclient/main.go:186]: Yep, exactly, now I see it's fixed

**Pull request merged**